topics/lua-api.md

```diff
@@ -306,33 +296,13 @@ Please refer to the [Data type conversion](#data-type-conversion) for more infor
 
 ### <a name="server.set_repl"></a> `server.set_repl(x)`
 
-* Since version: 3.2.0
 * Available in scripts: yes
 * Available in functions: no
 
 **Note:**
-this feature is only available when script effects replication is employed.
-Calling it when using verbatim script replication will result in an error.
-As of Redis OSS version 2.6.0, scripts were replicated verbatim, meaning that the scripts' source code was sent for execution by replicas and stored in the AOF.
-An alternative replication mode added in version 3.2.0 allows replicating only the scripts' effects.
-As of Redis OSS version 7.0, script replication is no longer supported, and the only replication mode available is script effects replication.
-
-**Warning:**
-this is an advanced feature. Misuse can cause damage by violating the contract that binds the Valkey master, its replicas, and AOF contents to hold the same logical content.
-
-This function allows a script to assert control over how its effects are propagated to replicas and the AOF afterward.
-A script's effects are the Valkey write commands that it calls.
-
-By default, all write commands that a script executes are replicated.
-Sometimes, however, better control over this behavior can be helpful.
-This can be the case, for example, when storing intermediate values in the master alone.
-
-Consider a script that intersects two sets and stores the result in a temporary key with `SUNIONSTORE`.
-It then picks five random elements (`SRANDMEMBER`) from the intersection and stores (`SADD`) them in another set.
-Finally, before returning, it deletes the temporary key that stores the intersection of the two source sets.
-
-In this case, only the new set with its five randomly-chosen elements needs to be replicated.
-Replicating the `SUNIONSTORE` command and the `DEL`ition of the temporary key is unnecessary and wasteful.
+Script replication is no longer supported.
+The only replication mode available is script effects replication.
+This feature is only available in Redis OSS versions prior to 7.0 which supported and used verbatim script replication by default.
```

Comment:
This is a little confusing: I couldn't parse what 'this' in the 3rd sentence, especially with the present tense. 

Maybe it should be something like: "Prior to Redis OSS 7.0, the default was verbatim script replication. After Redis OSS 7.0 (and Valkey), script effects replication is only replication mode available."

Reply:
Yeah, I see now the 3rd sentence contradicts the other sentences. I'll change it.